bot/decorators.py

```diff
@@ -3,15 +3,21 @@
 from discord.ext.commands import Context
 
 
+def invoke_with_prefix(prefix: str) -> commands.check:
+    '''Allow a command to only be invoked with one prefix
```

Comment:
We use double quotes instead of single quotes, consistency would be nice :P

Reply:
I'm consistent with PEP8 but it's ok, I can change the quotes.

Reply:
please do.